src/powerflow_method.jl

```diff
@@ -370,34 +383,85 @@ function _newton_powerflow(
     return false
 end
 
+"""If initial residual is large, run a DC power flow and see if that gives
+a better starting point for angles. Return the original or the result of the DC powerflow,
+whichever gives the smaller residual."""
+function improve_x0!(x0::Vector{Float64},
```

Comment:
We should have some dispatch mechanism here that can be used to decided which the initial guess improvement method